### PR TITLE
Old Washoe Club

### PR DESCRIPTION
**Pull request by anthony-colpron**

### CODE REVIEW

File: server/game/cards/12-2T2D/OldWashoeClub.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            }
+        });
+        this.action({
+            title: 'Controller Shootout, Boot: Discard a Sidekick.',
+            cost: ability.costs.bootSelf(),
+            playType: ['shootout'],
+            target: {
+                activePromptTitle: 'Choose a sidekick',
+                choosingPlayer: 'current',
+                cardCondition: {
```
Comment (anthony-colpron) [inline server/game/cards/12-2T2D/OldWashoeClub.js:34]: Doesn't have to be opponent's sidekick

File: server/game/cards/12-2T2D/OldWashoeClub.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
+const DeedCard = require('../../deedcard.js');
+const GameActions = require('../../GameActions/index.js');
+
+class OldWashoeClub extends DeedCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Controller Shootout, Boot: Send an opposing A-value dude home booted.',
```
Comment (mmeldo) [inline server/game/cards/12-2T2D/OldWashoeClub.js:7]: I would use shorter titles. Player should know the card they are using, so it should just help him to be sure what ability he is triggering.
For example: 
`Shootout: Send A-dude home booted`
`Shootout: Discard a Sidekick`

**Review by mmeldo — CHANGES_REQUESTED** [review]
Nice, your first PR and I see you are already up to speed. Just suggested a small change to the ability titles.

File: server/game/cards/12-2T2D/OldWashoeClub.js (hunk 2 above)
Comment (anthony-colpron) [reply server/game/cards/12-2T2D/OldWashoeClub.js]: Done

**Review by mmeldo — APPROVED** [review]

**Pull request merged**